### PR TITLE
Improved git protocol guide

### PR DESCRIPTION
**Pull request by colmarius**
Document steps both with command line and SourceTree GUI.
Add table of contents section.

### CODE REVIEW

**Comment by colmarius** [comment]
@BuddyBroker/developers Hi team, here we update our git protocol for both SourceTree and command line. Let me know what you think: if something is missing or not clear.

**Comment by colmarius** [comment]
@MagdaMalinowska done in https://github.com/BuddyBroker/guides/commit/2b8765a166e8a22e673a8ec22426454e4889b144

**Comment by andrzejtrzaska** [comment]
Looks good.

**Comment by colmarius** [comment]
@AniaCodequest I added in https://github.com/BuddyBroker/guides/commit/5a3caf08202d0943318a936d19eb99cac7052f35 advice to prefer atomic commits.

**Comment by andrzejtrzaska** [comment]
approved.

**Comment by AniaCodequest** [comment]
Looks good!

File: protocol/git/README.md [hunk 1]
```diff
@@ -20,23 +31,55 @@ Write a Feature
...
-    git status
-    git commit --verbose
+- command line:
+
+        git status
+        git commit --verbose
+
+- SourceTree:
+
+    + select and inspect "Working Copy"
```
Comment (MagdaMalinowska) [inline protocol/git/README.md]: Same view is for 'Commit' button from top menu

Comment (colmarius) [inline protocol/git/README.md]: Done in https://github.com/BuddyBroker/guides/commit/f23213ea34bde4bd4bff716523e15ea146f02270

**Comment by MagdaMalinowska** [comment]
:+1:

**Comment by colmarius** [comment]
Thanks everyone. We agree on all aspects. Merging PR.

**Pull request merged**